packages/react-reconciler/src/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -745,7 +745,7 @@ describe('ReactIncrementalErrorHandling', () => {
     expect(ReactNoop.flush).toWarnDev(
       'Warning: React.createElement: type is invalid -- expected a string',
     );
-    expect(ReactNoop.getChildren()).toEqual([
+    expect(ReactNoop.getChildren()).toContent([
```

Comment:
I don't think we should add a custom matcher just for this one use case.

I'm sorry I wasn't clear before—when I said about custom matchers I meant about changing the code that *currently* uses `toThrow()`, not this use case.

Here, you can just use `toContain` and remove the part with the stack trace (it's not essential to this test)

Reply:
`toContain` is checking if something is within an array with `===`. The `span()` method returns an object which would fail the strict equals. Could I keep it as `.toEqual()` and strip out the stack trace? I'll get to work on a code sample for this right now.

Reply:
You can do `expect(ReactNoop.getChildren()[0].children[0]).toContain(`

Reply:
In these instances it seems as though `children` is an empty array. I'm getting: `Expected array but received undefined` when I use that line in the `expect()` statement.

Reply:
Well, you can inspect the structure and figure out which one to compare 😉